lib/manager.js

```diff
@@ -0,0 +1,71 @@
+const path = require('path')
+const os = require('os')
+var Spinner = require('cli-spinner').Spinner
+const { Worker } = require('./worker_threads')
+
+const spinner = new Spinner('running... %s')
+spinner.setSpinnerString('|/-\\')
+
+const numWorkers = Math.max(Math.floor(os.cpus().length * 0.75), 1)
```

Comment:
This should be an option/parameter, not decided based on the available CPUs. This should also be printed by the CLI.